### ISSUE TITLE
Missing System.Composition.Convention

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#6656]
Build warning:
```
Could not resolve this reference. Could not locate the assembly "System.Composition.Convention". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.
```

https://www.nuget.org/packages/System.Composition.Convention/

> VS bug [#734934](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/734934)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-7] Comment by **xamarinc**



Fixed in version 7.8.0.1306 (master)



Author: Marius Ungureanu  
 Commit: a41d3838b309dbe4bdadc5c6da38d13e28c0ac99 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-9] Comment by **xamarinc**



Fixed in version 8.0.0.800 (release-8.0)



Author: Marius Ungureanu  
 Commit: a41d3838b309dbe4bdadc5c6da38d13e28c0ac99 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-10] Comment by **xamarinc**



Fixed in version 7.8.0.1339 (release-7.8)



Author: Marius Ungureanu  
 Commit: a41d3838b309dbe4bdadc5c6da38d13e28c0ac99 (mono/monodevelop)
